ddtrace/contrib/aiohttp/patch.py

```diff
@@ -1,25 +1,130 @@
+import os
+
+from yarl import URL
+
 from ddtrace import config
 from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils import get_argument_value
+from ddtrace.internal.utils.formats import asbool
+from ddtrace.vendor import wrapt
 
+from ...ext import SpanTypes
+from ...pin import Pin
+from ...propagation.http import HTTPPropagator
+from ..async_trace_utils import with_traced_module
+from ..trace_utils import ext_service
+from ..trace_utils import set_http_meta
+from ..trace_utils import unwrap
+from ..trace_utils import with_traced_module as with_traced_module_sync
+from ..trace_utils import wrap
 
-log = get_logger(__name__)
 
+log = get_logger(__name__)
 
 config._add(
-    "aiohttp",
+    "aiohttp_client",
     dict(
-        distributed_tracing=True,
+        distributed_tracing_enabled=asbool(os.getenv("DD_AIOHTTP_CLIENT_DISTRIBUTED_TRACING", False)),
     ),
 )
 
 
+class _WrappedConnectorClass(wrapt.ObjectProxy):
+    def __init__(self, obj, pin):
+        super().__init__(obj)
+        pin.onto(self)
+
+    async def connect(self, req, *args, **kwargs):
+        pin = Pin.get_from(self)
+        with pin.tracer.trace("%s.connect" % self.__class__.__name__):
+            result = await self.__wrapped__.connect(req, *args, **kwargs)
+            return result
+
+    async def _create_connection(self, req, *args, **kwargs):
+        pin = Pin.get_from(self)
+        with pin.tracer.trace("%s._create_connection" % self.__class__.__name__):
+            result = await self.__wrapped__._create_connection(req, *args, **kwargs)
+            return result
+
+
+@with_traced_module
+async def _traced_clientsession_request(aiohttp, pin, func, instance, args, kwargs):
+    method: str = get_argument_value(args, kwargs, 0, "method")
```

Comment:
```suggestion
    method = get_argument_value(args, kwargs, 0, "method")  # type: str
```

?

I know this only works on newer Python, but still should we prefer comment type annotations?